airbyte-ci/connectors/pipelines/pipelines/bases.py

```diff
@@ -551,6 +556,10 @@ def post_comment_on_pr(self) -> None:
         ]
         markdown_comment += tabulate(report_data, headers=["Step", "Result"], tablefmt="pipe") + "\n\n"
         markdown_comment += f"🔗 [View the logs here]({self.html_report_url})\n\n"
+
+        if self.pipeline_context.dagger_cloud_url:
+            markdown_comment += f"☁️ [View commit in Dagger Cloud]({self.pipeline_context.dagger_cloud_url})\n\n"
```

Comment:
```suggestion
            markdown_comment += f"☁️ [View runs for commit in Dagger Cloud]({self.pipeline_context.dagger_cloud_url})\n\n"
```

Reply:
done in [9ab0f68](https://github.com/airbytehq/airbyte/pull/28947/commits/9ab0f680bea0d851a62a4c2a0bfd530d222dd110)